README.md

```diff
@@ -1,10 +1,18 @@
 # Software Engineering: Homework-2b
 
-Write a "good" Github Repo
-
 ## Description
 
-To be submitted as homework assignment HW-2b for the course Software Engineering (CSC-510) under Professor Timothy Menzies.
+The primary purpose of this repository was to get familiar with the various features of Github and to concisely integrate them. This repository was created as a homework assignment (HW-2b) for our Software Engineering (CSC-510) course under [Prof. Timothy Menzies](http://menzies.us/).
+
+We have included a simple `src/validate.py` file which consists of a class `Calculator` consisting of methods performing certain list-based manipulations. We have integrated this `src` directory with our `setup.py`. This file not only eases installation but provides functionality to integrate testing using Pytest. As can be observed in [`main.yml`](https://github.com/SN-18/SE-Homework-2b/blob/main/.github/workflows/main.yml), our argument passed to the Github Actions workflow is `python setup.py develop && pytest` which runs the tests in `./tests/` directory.
+
+Apart from this, we have ensured that the following practices are included while creating this repository -
+1. [Adding a Kanban style project board](https://github.com/SN-18/SE-Homework-2b/projects/2)
+2. Creating Github Issues for proposals/bugs
+3. Creating branches/forks and submitting Pull Requests (PRs)
+4. These PRs are being reviewed before getting commited
+5. Adding descriptive badges
+6. Providing an appropriate `LICENSE`, `CODE_OF_CONDUCT`, and `CONTRIBUTING` file with `CITATION.cff`
```

Comment:
Minor change, it should be:
"Providing an appropriate 'LICENSE', 'CODE_OF_CONDUCT', 'CONTRIBUTING'  and 'CITATION' files."

The citation.cff is standalone for the whole repo, it is integrated with zenodo. It is not just linked to CONTRIBUTING file.